mk/repos.mk

```diff
@@ -0,0 +1,12 @@
+.PHONY: import-repos
+TMPDIR := $(shell mktemp -d)
+
+.PHONY: repos-download
+repos-download: ## Download external repo urls from Image Builder
```

Comment:
I would add to the rule comment that it generates the file `pkg/external_repos/external_repos.json`